src/lib/get-messages.js

```diff
@@ -2,7 +2,7 @@ import assign from 'object-assign';
 import locales from '../../public/locales.json';
 
 function getMessages(locale) {
-  if (locale === 'es-MX') {
+  if (['es-AR', 'es-CL', 'es-MX'].indexOf(locale) >= 0) {
```

Comment:
Should we add es-AR and es-CL empty locale folders with a `.keep` file to the repo to expose these locales?